scripts/linters/warranted_safe_pipe_usages.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""List of warranted usages of safe pipe in the code."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # isort:skip pylint: disable=import-only-modules
+
+# Contains the dict of all files that use safe pipe along with the usage count.
+# Please keep the list in alphabetical order.
+# NOTE TO DEVELOPERS: do not add any new files to this list without asking
+# @srijanreddy98 or @bansalnitish first.
+# Format: {
+#   file name: list of occurrences of safe pipe in the file.
+# }
+WARRANTED_USAGES = {
```

Comment:
Maybe declare this variable at the top of html_linter.py file.

Reply:
I thought I would do that too, but I and Nitish need to be notified when someone adds a new occurrence to the list so the best to do that would be to add it to a new file.

Reply:
Okay, It's fine then